src/live.asp.net/Views/Shared/_YouTubeShow.cshtml

```diff
@@ -20,7 +20,7 @@
                     <span class="show-date">@Model.ShowDate.DateTime.ToString("MMM dd, yyyy")</span>
                     @if (Model.IsNew)
                     {
-                        <span class="label label-success">New</span>
+                        <span class="label label-success" style="font-size:14px">New</span>
```

Comment:
Please don't use inline styles. Make an appropriate change to the `site.css` file and markup if necessary.